README.md

```diff
@@ -347,6 +347,7 @@ Tools to manage tabular data files, such as CSV, spreadsheets, and database tabl
 
 * [csvkit](https://github.com/wireservice/csvkit) - A suite of command-line tools for converting to and working with CSV, the king of tabular file formats.
 * [csvq](https://github.com/mithrandie/csvq) - SQL-like query language for csv.
+* [csvlens](https://github.com/YS-L/csvlens) - Command line CSV file viewer; like `less` but made for CSV.
```

Comment:
```suggestion
* [csvlens](https://github.com/YS-L/csvlens) - CSV file viewer; like `less` but made for CSV.
```
IMO don't need the "Command line" part because this repo is just for CLI tools ;)